open-metadata-implementation/access-services/data-platform/data-platform-api/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/events/DataViewElement.java

```diff
@@ -0,0 +1,82 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Similar to previous comment, is this used?

Reply:
True. Removed.